portal-api/src/v1/users/controller.test.js

```diff
@@ -48,4 +48,32 @@ describe('user controller', () => {
       expect(mockCallback).toHaveBeenCalledTimes(1);
     });
   });
+
+  describe('createPasswordToken', () => {
```

Comment:
added tests to check when trying to reset password that the user isn't blocked or disabled, if they are then we don't return a token

Reply:
Test name makes sense but any textual explanation is always helpful when being visited in fresh.